lib/class-wp-theme-json.php

```diff
@@ -271,25 +250,43 @@ class WP_Theme_JSON {
 		),
 	);
 
+	const ELEMENTS = array(
+		'link' => 'a',
```

Comment:
Making link color an element shoulds like a good idea but I wonder what will happen when the link color is applied as an inline style of the block? Will we still use a CSS variable for that case?

Reply:
> I wonder what will happen when the link color is applied as an inline style of the block? Will we still use a CSS variable for that case?

I think personally we should generate the exact same style global styles generate (probably server side) and avoid the CSS variable entirely.

Reply:
related https://github.com/WordPress/gutenberg/pull/30541#discussion_r621995913